support/openharmony/build-profile.json5

```diff
@@ -4,16 +4,16 @@
       {
         "name": "default",
         "signingConfig": "default",
-        "compileSdkVersion": 12,
-        "compatibleSdkVersion": 12,
-        "targetSdkVersion": 12,
+        "compileSdkVersion": 15,
+        "compatibleSdkVersion": 15,
+        "targetSdkVersion": 15,
         "runtimeOS": "OpenHarmony"
       },
       {
         "name": "harmonyos",
-        "signingConfig": "hos",
-        "compatibleSdkVersion": "5.0.0(12)",
-        "targetSdkVersion": "5.0.0(12)",
+        "signingConfig": "default",
```

Comment:
The signing config here can't be the same name as the OpenHarmony one above (OH and HOS require different signing configurations). In your signing-configs.json you can just change the "name" field to `hos` instead.